Yona/Yona/CustomCells/YonaUserTableViewCell.swift

```diff
@@ -116,18 +104,26 @@ class YonaUserTableViewCell: PKSwipeTableViewCell {
         
         let formatter = DateFormatter()
         formatter.dateFormat = "HH:mm"
-        
         if aMessage.messageType == notificationType.GoalConflictMessage {
             boldLineLabel.text = aMessage.simpleDescription() + " " + aMessage.activityTypeName + " - " + formatter.string(from: aMessage.creationTime as Date)
+            avatarImageView.image = R.image.adultSad()
         } else {
             boldLineLabel.text = aMessage.simpleDescription()
+            if let link = aMessage.userPhotoLink,
+                let URL = URL(string: link) {
+                avatarImageView.kf.setImage(with: URL)
+            } else {
+                avatarImageView.image = nil
+            }
         }
+        
         normalLineLabel.text = "\(aMessage.nickname)"
      
         var tmpnickname = ""
         tmpnickname = aMessage.nickname
```

Comment:
Can't you remove ``tmpnickname`` and test on ``aMessage.nickname`` directly?.

Reply:
Yes, I can remove and compare directly. Updated code accordingly.